src/components/AgreementFilters/AgreementFilters.js

```diff
@@ -235,12 +238,32 @@ export default class AgreementFilters extends React.Component {
     />;
   }
 
+  renderStartDateFilter = () => {
+    return <DateFilter
+      activeFilters={this.props.activeFilters}
+      filterHandlers={this.props.filterHandlers}
+      name="startDate"
+    />;
+  }
+
+  renderEndDateFilter = () => {
+    return <DateFilter
+      activeFilters={this.props.activeFilters}
+      allowNull
+      filterHandlers={this.props.filterHandlers}
+      name="endDate"
+      resourceName={this.props.intl.formatMessage({ id: 'ui-agreements.agreements.lowerCase' })}
```

Comment:
Cant we use `FormattedMessage` here?

Reply:
I don't think so. I need to pass a string rather than an object. FormattedMessage returns a React node

Reply:
I'd rather use the hook than the messier HOC, but I didn't want to convert the entire filter component to functional

Reply:
The easiest alternative I suppose would be a prop something like `noDateCheckboxLabel`, where we pass the entire label translation, removing that from the stripes-erm-components component

Reply:
But that does seem overkill compared to passing "agreements" and "licenses" as strings to a ready made translation in the DateFilter comp as values

Reply:
Lets make use of this opportunity to tweak that component to a functional component and bring in the `useIntl` hook

Reply:
Done